tests/product/tests/test_file_system_rules.py

```diff
@@ -34,13 +34,14 @@
      *fs_tc.cis_1_1_16,
      *fs_tc.cis_1_1_17,
      *fs_tc.cis_1_1_18,
-     *skip_param_case(skip_list=fs_tc.cis_1_1_19[0:3],
-                      data_to_report=SkipReportData(
-                          url_title="security-team: #4484",
-                          url_link="https://github.com/elastic/security-team/issues/4484",
-                          skip_reason="known issue: flaky file_system_rules tests"
-                      )),
-     *fs_tc.cis_1_1_19[3:],
+     *fs_tc.cis_1_1_19,
+    #  *skip_param_case(skip_list=fs_tc.cis_1_1_19[0:3],
+    #                   data_to_report=SkipReportData(
+    #                       url_title="security-team: #4484",
+    #                       url_link="https://github.com/elastic/security-team/issues/4484",
+    #                       skip_reason="known issue: flaky file_system_rules tests"
+    #                   )),
+    #  *fs_tc.cis_1_1_19[3:],
```

Comment:
is it skipped? why are we commenting out this part?

Reply:
PR wasn't ready.